### DISCUSSION TITLE
Unable to connect git over https using Bitbucket username and app password

### DISCUSSION THREAD

**Discussion by Prashanth073-cloud** [argoproj/argo-cd#8660] [Q&A]
Since today, Bitbucket has stopped authentication over HTTPS using password and informed to use app passwords instead.  Consequently, my ArgoCD stopped working and showing authentication error. So I tried creating a repository using "Connect repo using HTTPS" from the ArgoCD UI and given my repository URL, username, app password but authentication is getting failed. I am getting the error "existing repository spec is different, use upsert flag to force update". Anyone facing the same issue?

**Comment by crenshaw-dev** [comment]
Can you open an issue, with a reproducible example a link to any Bitbucket change announcement you've seen?

**Reply by Prashanth073-cloud** [reply]
Thank you @crenshaw-dev  for your reply. I found the solution and thought of deleting the discussion before anyone comments on it but couldn't do it. I have deleted the already existed repository in Argo-CD and then tried creating it again. It worked. Following link has the information about the Bitbucket announcement.
[https://support.atlassian.com/bitbucket-cloud/docs/app-passwords/](url)
